.hooks/coverage-collect.js

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/node
```

Comment:
should this be `	#!/usr/bin/env node`?

Reply:
Good point! Done and updated the other script as well.